docs/src/content/docs/hi/environmental-impact.md

```diff
@@ -89,20 +89,21 @@ JavaScript को पार्स करना और संकलित कर
 क्या आप जानना चाहते हैं कि अन्य दस्तावेज़ीकरण फ्रेमवर्क की तुलना कैसे की जाती है?
 [Website Carbon Calculator][wcc] के साथ ये परीक्षण विभिन्न उपकरणों के साथ बनाए गए समान पृष्ठों की तुलना करते हैं।
 
-| फ्रेमवर्क                   | प्रति पृष्ठ विज़िट CO₂ |
-| --------------------------- | ---------------------- |
-| [Starlight][sl-carbon]      | 0.01g                  |
-| [VitePress][vp-carbon]      | 0.05g                  |
-| [Docus][dc-carbon]          | 0.05g                  |
-| [Sphinx][sx-carbon]         | 0.07g                  |
-| [MkDocs][mk-carbon]         | 0.10g                  |
-| [Nextra][nx-carbon]         | 0.11g                  |
-| [docsify][dy-carbon]        | 0.11g                  |
-| [Docusaurus][ds-carbon]     | 0.24g                  |
-| [Read the Docs][rtd-carbon] | 0.24g                  |
-| [GitBook][gb-carbon]        | 0.71g                  |
-
-<small>डेटा 14 मई 2023 को एकत्र किया गया। नवीनतम आंकड़े देखने के लिए लिंक पर क्लिक करें।</small>
+| फ्रेमवर्क                   | प्रति पृष्ठ विज़िट CO₂ | रेटिंग |
+| --------------------------- | ------------------ | :----: |
+| [Starlight][sl-carbon]      | 0.01g              |   A+   |
+| [Read the Docs][rtd-carbon] | 0.05g              |   A+   |
+| [Sphinx][sx-carbon]         | 0.06g              |   A+   |
+| [VitePress][vp-carbon]      | 0.07g              |   A+   |
+| [Docus][dc-carbon]          | 0.09g              |   A+   |
+| [docsify][dy-carbon]        | 0.10g              |   A    |
+| [Nextra][nx-carbon]         | 0.11g              |   A    |
+| [MkDocs][mk-carbon]         | 0.19g              |   B    |
+| [Docusaurus][ds-carbon]     | 0.21g              |   B    |
+| [GitBook][gb-carbon]        | 0.43g              |   C    |
+| [Mintlify][mt-carbon]       | 1.22g              |   F    |
+
+<small>डेटा 22 July 2024 को एकत्र किया गया। नवीनतम आंकड़े देखने के लिए लिंक पर क्लिक करें।</small>
```

Comment:
Should "July" be translated? Looks like the previous month was so I assume it should be too? Any thought?

Reply:
absolutely it should be translated 😄 . thank you again for reviewing this one